server/backend/testspg/crud/test_idim_proxy_service.py

```diff
@@ -73,4 +81,32 @@ def test_idir_search_user_not_exist_no_user_found(self):
         search_params.userId = not_exists_idir_user
         search_result = idim_proxy_api.search_idir(search_params)
 
-        assert search_result['found'] == False
+        assert search_result["found"] == False
+
+    @pytest.mark.skip(
+        reason="need idir user guid to run this test, switch to use bceid search bceid later"
```

Comment:
skip the bceid test, as it requires IDIR guid currently, could change to use Bceid guid for testing later